v3.3.0 で起動に失敗する

Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        | Ubuntu 18.04 |
|GROWI     |3.3.0|
|node.js   ||
|npm       ||
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

<!--
*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*


### Client

| item     | version |
| ---      | --- |
|OS        ||
|browser   |x.y.z|
-->


How to reproduce? (再現手順)
---------------------------

1. [こちら](https://github.com/weseek/growi-docker-compose#upgrade)に記載の方法で 3.2.10 から 3.3.0 へ更新を行う
1. `docker-compose up` で起動する

What happens? (症状)
---------------------

- 以下のエラーが発生し、app コンテナが起動できない  
  （app のみ抜粋、mongodb および elasticsearch は起動できる）
    ```
    app_1            | 2018/12/18 19:31:10 Waiting for: tcp://mongo:27017
    app_1            | 2018/12/18 19:31:10 Waiting for: tcp://elasticsearch:9200
    app_1            | 2018/12/18 19:31:10 Problem with dial: dial tcp 172.19.0.2:9200: getsockopt: connection refused. Sleeping 1s
    app_1            | 2018/12/18 19:31:10 Connected to tcp://mongo:27017
    app_1            | 2018/12/18 19:31:11 Problem with dial: dial tcp 172.19.0.2:9200: getsockopt: connection refused. Sleeping 1s
    app_1            | 2018/12/18 19:31:12 Problem with dial: dial tcp 172.19.0.2:9200: getsockopt: connection refused. Sleeping 1s
    app_1            | 2018/12/18 19:31:13 Problem with dial: dial tcp 172.19.0.2:9200: getsockopt: connection refused. Sleeping 1s
    app_1            | 2018/12/18 19:31:14 Problem with dial: dial tcp 172.19.0.2:9200: getsockopt: connection refused. Sleeping 1s
    app_1            | 2018/12/18 19:31:15 Problem with dial: dial tcp 172.19.0.2:9200: getsockopt: connection refused. Sleeping 1s
    app_1            | 2018/12/18 19:31:16 Problem with dial: dial tcp 172.19.0.2:9200: getsockopt: connection refused. Sleeping 1s
    app_1            | 2018/12/18 19:31:17 Connected to tcp://elasticsearch:9200
    app_1            |
    app_1            | > growi@3.3.0 preserver:prod /opt/growi
    app_1            | > npm run migrate
    app_1            |
    app_1            |
    app_1            | > growi@3.3.0 migrate /opt/growi
    app_1            | > npm run migrate:up
    app_1            |
    app_1            |
    app_1            | > growi@3.3.0 migrate:up /opt/growi
    app_1            | > migrate-mongo up -f config/migrate.js
    app_1            |
    app_1            |
    app_1            | > growi@3.3.0 server:prod /opt/growi
    app_1            | > env-cmd config/env.prod.js node src/server/app.js
    app_1            |
    app_1            | module.js:550
    app_1            |     throw err;
    app_1            |     ^
    app_1            |
    app_1            | Error: Cannot find module 'url-join'
    app_1            |     at Function.Module._resolveFilename (module.js:548:15)
    app_1            |     at Function.Module._resolveFilename (/opt/growi/node_modules/module-alias/index.js:43:29)
    app_1            |     at Function.Module._load (module.js:475:25)
    app_1            |     at Module.require (module.js:597:17)
    app_1            |     at require (internal/module.js:11:18)
    app_1            |     at Object.<anonymous> (/opt/growi/src/lib/service/cdn-resources-service.js:2:17)
    app_1            |     at Module._compile (module.js:653:30)
    app_1            |     at Object.Module._extensions..js (module.js:664:10)
    app_1            |     at Module.load (module.js:566:32)
    app_1            |     at tryModuleLoad (module.js:506:12)
    app_1            | npm ERR! code ELIFECYCLE
    app_1            | npm ERR! errno 1
    app_1            | npm ERR! growi@3.3.0 server:prod: `env-cmd config/env.prod.js node src/server/app.js`
    app_1            | npm ERR! Exit status 1
    app_1            | npm ERR!
    app_1            | npm ERR! Failed at the growi@3.3.0 server:prod script.
    app_1            | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
    app_1            |
    app_1            | npm ERR! A complete log of this run can be found in:
    app_1            | npm ERR!     /root/.npm/_logs/2018-12-18T19_31_19_090Z-debug.log
    app_1            | 2018/12/18 19:31:19 Command exited with error: exit status 1
    growi_app_1 exited with code 1
        ```



What is the expected result? (期待される動作)
-------------------------------------------

- 正しく起動できる



Note
--------

- 3.2.10 へ戻した場合は正しく起動できる。
- Dockerfile でバージョンを指定した以外は、特に設定の変更は行っていない。

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,7 +1,11 @@
 CHANGES
 ========
 
-## 3.3.0-RC
+## 3.3.1-RC
+
+* Fix: Missing dependencies for production (url-join)
+
+## 3.3.0
 
 * Feature: NO_CDN Mode
 * Feature: Add option to show/hide restricted pages in list

--- a/package.json
+++ b/package.json
@@ -117,6 +117,7 @@
     "string-width": "^2.1.1",
     "swig-templates": "^2.0.2",
     "uglifycss": "^0.0.29",
+    "url-join": "^4.0.0",
     "xss": "^1.0.3"
   },
   "devDependencies": {
@@ -203,7 +204,6 @@
     "throttle-debounce": "^2.0.0",
     "toastr": "^2.1.2",
     "uglifyjs-webpack-plugin": "^2.0.1",
-    "url-join": "^4.0.0",
     "webpack": "^4.12.0",
     "webpack-assets-manifest": "^3.0.1",
     "webpack-bundle-analyzer": "^3.0.2",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "growi",
-  "version": "3.3.0-RC",
+  "version": "3.3.1-RC",
   "description": "Team collaboration software using markdown",
   "tags": [
     "wiki",

